eng/pipelines/coreclr/templates/xplat-pipeline-job.yml

```diff
@@ -119,4 +119,11 @@ jobs:
     - ${{ each variable in parameters.variables }}:
       - ${{insert}}: ${{ variable }}
 
+    - name: osArg
+      value: ''
+
+    - ${{ if eq(parameters.osGroup, 'FreeBSD') }}:
+      - name: osArg
```

Comment:
You still need to flow this to the coreclr build scripts in coreclr/templates/build-job.yml and coreclr/templates/corelib-build-job.yml.

Reply:
ok. The container is not quite right and I'm still investigating.